src/Entity/BaseCategory.php

```diff
@@ -13,15 +13,8 @@
 
 namespace Sonata\ClassificationBundle\Entity;
 
-use Doctrine\ORM\PersistentCollection;
 use Sonata\ClassificationBundle\Model\Category as ModelCategory;
 
 abstract class BaseCategory extends ModelCategory
 {
-    public function disableChildrenLazyLoading(): void
```

Comment:
This method was not deprecated. We need a CHANGELOG entry for this

Reply:
Done on 3.x instead https://github.com/sonata-project/SonataClassificationBundle/pull/765